grabba_grabba_hey/sentinel_downloader.py

```diff
@@ -171,69 +173,91 @@ def parse_xml(xml):
     # for x in img.getchildren():
 
 
-def download_sentinel(location, input_start_date, input_sensor, output_dir,
-                      input_end_date=None, username="guest", password="guest"):
-    input_sensor = input_sensor.upper()
-    sensor_list = ["S1", "S2"]
-    if not input_sensor in sensor_list:
-        raise ValueError("Sensor can only be S1 or S2. You provided %s"
-                         % input_sensor)
+def download_sentinel(location, input_start_date, 
```

Comment:
Bummer, maybe I didn't understand: did you extend the previous version that gets S1 and S2 to also do S3 as well, or does this version only do S3 (no probs either way, just want clarification)